### ISSUE TITLE
Dynamic tracking interval for each series

### ISSUE THREAD

**Issue by DakuTree** [DakuTree/manga-tracker#6]
Some series only update once a week, some only once a month, and others are still trying to catch-up with the latest chapter. It would be nice to have different interval times depending on the series.

A small idea to add on to this would be some kind of logging for latest_chapter changes, so we can see when series actually update.

**Comment by DakuTree** [comment]
8 months later, I'm not too sure if this is actually needed anymore.

Since we moved over to using the custom updater for MangaFox and Batoto, the amount of updates we have to do has been severely reduced.  
Custom series are only updated every 36 hours (compared to 12~), and only if they weren't updated via the custom updater.  
Now although this could be useful for other non-custom updater sites, I'm not sure if we're following enough on other sites to justify adding this.  

Just from a quick look, we only have 376 of the 1902 series which isn't covered by the custom updater. Of those, I'd say 192 series could be added to the custom updater (MangaHere (107), DynastyScans? (67), WebToons (18)), meaning we're left with 184 (Less than 10%).  

I really think with the addition of the custom updater this feature has become somewhat useless.  
Might be useful much later down the line, and if so I'll re-open this, but for now I'll be closing it.
